frontend/src/features/map_planning/components/EditorMap.tsx

```diff
@@ -320,13 +329,20 @@ export const EditorMap = ({ layers }: MapProps) => {
               stageMouseWheelListeners,
               stageClickListeners,
             }}
+            selectable={isShapeSelectionEnabled}
           >
             <BaseLayer
               stageListenerRegister={baseStageListenerRegister}
               opacity={layersState.base.opacity}
               visible={layersState.base.visible}
               listening={getSelectedLayerType() === LayerType.Base}
             />
+            <DrawingLayer
+              stageListenerRegister={baseStageListenerRegister}
```

Comment:
Please don't use the listener register!

It was added because I did not know that we were supposed to use stageRef from the map store and will be removed in the near future.
Look at the following example from PlantsLayer.tsx to see how global stage listeners should be registered instead
```js
  useEffect(() => {
    if (!listening) {
      return;
    }
    const transformerActions = useTransformerStore.getState().actions;

    useMapStore.getState().stageRef.current?.on('click.placePlant', handleCreatePlanting);
    useMapStore.getState().stageRef.current?.on('click.unselectPlanting', handleUnselectPlanting);
    useMapStore.getState().stageRef.current?.on('mouseup.selectPlanting', handleSelectPlanting);
    transformerActions.addEventListener('transformend.plants', handleTransformPlanting);
    transformerActions.addEventListener('dragend.plants', handleMovePlanting);

    return () => {
      useMapStore.getState().stageRef.current?.off('click.placePlant');
      useMapStore.getState().stageRef.current?.off('click.unselectPlanting');
      useMapStore.getState().stageRef.current?.off('mouseup.selectPlanting');
      transformerActions.removeEventListener('transformend.plants');
      transformerActions.removeEventListener('dragend.plants');
    };
  }, [
    listening,
    handleCreatePlanting,
    handleTransformPlanting,
    handleMovePlanting,
    handleUnselectPlanting,
    handleSelectPlanting,
  ]);
```